kola/cluster/torcx_manifest.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type TorcxManifest struct {
+	Packages []TorcxPackage
+}
+
+// a copy of the TorcxManifest type with no UnmarshalJSON method so the below doesn't do a round of recursion
+type torcxManifestUnmarshalValue struct {
+	Packages []TorcxPackage
+}
+
+func (t *TorcxManifest) UnmarshalJSON(b []byte) error {
```

Comment:
If there were to be a user who wanted to json marshal it, we could implement a `MarshalJSON` which re-added the wrapping and version to let it roundtrip.

Since this struct is meant to be the useable representation of that json format, I don't think it needs to be a detail.

Reply:
After thinking about it more, it does seem like it fits better as a global in `kola`, even though I really don't like package globals..